src/Illuminate/Validation/Rules/Password.php

```diff
@@ -279,6 +290,50 @@ public function rules($rules)
         return $this;
     }
 
+    /**
+     * Create a password that adheres to the current validation rules.
+     *
+     * @param  int  $length
+     * @return string
+     */
+    public function create(int $length = 8): string
+    {
+        $length = ($length >= $this->min) ? $length : random_int($this->min, $this->min + 10);
+
+        $password = [];
+
+        $upper = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'X', 'Y', 'Z'];
```

Comment:
You can generate uppercase letters like

```php
$upper = array_reduce(
            range('A', 'Z'),
            fn ($p, $n) => $p.$n
        );
```

Reply:
@ankurk91 - Fair enough. I'm not sure it really matters in the grand scheme of things, but I believe you just need `range('A', 'Z')` to generate the array.

Reply:
Works for lowercase and numbers as well.

```
range('a', 'z');
range(0, 9);
```